src/routes/+layout.ts

```diff
@@ -2,6 +2,6 @@ import type { LayoutLoad } from './$types';
 import { getSupabase } from '@supabase/auth-helpers-sveltekit';
 
 export const load: LayoutLoad = async (event) => {
-	const { session } = await getSupabase(event);
-	return { session };
+	const { session, supabaseClient } = await getSupabase(event);
+	return { session, supabaseClient };
```

Comment:
Is it necessary to have `supabaseClient` sent as layout data when we can import it directly from `$lib/auth` ?